packages/automation-contracts/scheduler/contracts/VestingSchedulerV2.sol

```diff
@@ -483,18 +442,18 @@ contract VestingSchedulerV2 is IVestingSchedulerV2, SuperAppBase {
             agg.sender,
             agg.receiver,
             schedule.cliffAndFlowDate,
-            0,
+            0, // flow rate
             schedule.cliffAmount,
-            0
+            totalVestedAmount - schedule.cliffAmount // flow delay compensation
         );
 
         emit VestingEndExecuted(
             agg.superToken,
             agg.sender,
             agg.receiver,
             schedule.endDate,
-            totalVestedAmount,
```

Comment:
Decided it's better to show the value under the "cliff and flow" event's "flow delay compensation" along with the "cliff amount".